src/resources/views/crud/fields/image.blade.php

```diff
@@ -198,7 +198,7 @@ function bpFieldInitCropperImageElement(element) {
                         });
                     } else {
 
-                        $(this).find("#remove").click(function() {
```

Comment:
You're right @carlobeltrame - this fix is independent from supporting SVGs and should definitely be done. Fortunately, from our conversation until now, it seems it's been fixed by a different PR 😄 So a ```composer update``` should get you the fixed file.